docs/docs/amoro.md

```diff
@@ -0,0 +1,89 @@
+**[Table Maintenance](https://amoro.apache.org/docs/latest/self-optimizing/)** features for a Lakehouse to provide out-of-the-box data warehouse experience, and helps data platforms or products easily build infra-decoupled, stream-and-batch-fused and lake-native architecture.
+**AMS(Amoro MetaStore)** is Amoro Management Service provides Lakehouse management features, like self-optimizing, data expiration, etc. It also provides a unified catalog service for all compute engines, which can also be combined with existing metadata services.
+
+# Auto Self-optimizing
+
+Lakehouse is characterized by its openness and loose coupling, with data and files maintained by users through various engines. While this
+architecture appears to be well-suited for T+1 scenarios, as more attention is paid to applying Lakehouse to streaming data warehouses and real-time
+analysis scenarios, challenges arise. For example:
+
+- Streaming writes bring a massive amount of fragment files
+- CDC ingestion and streaming updates generate excessive redundant data
+- Using the new data lake format leads to orphan files and expired snapshots.
+
+These issues can significantly affect the performance and cost of data analysis. Therefore, Amoro has introduced a Self-optimizing mechanism to
+create an out-of-the-box Streaming Lakehouse management service that is as user-friendly as a traditional database or data warehouse. Self-optimizing involves various procedures such as file compaction, deduplication, and sorting.
+
+The architecture and working mechanism of Self-optimizing are shown in the figure below:
+
+![Self-optimizing architecture](https://github.com/apache/amoro/blob/master/docs/images/concepts/self-optimizing_arch.png)
+
+The Optimizer is a component responsible for executing Self-optimizing tasks. It is a resident process managed by AMS(Amoro MetaStore). AMS is responsible for
+detecting and planning Self-optimizing tasks for tables, and then scheduling them to Optimizers for distributed execution in real-time. Finally, AMS
+is responsible for submitting the optimizing results. Amoro achieves physical isolation of Optimizers through the Optimizer Group.
+
+The core features of [Amoro's Self-optimizing](https://amoro.apache.org/docs/latest/self-optimizing/) are:
+
+- Automated, Asynchronous and Transparent — Continuous background detecting of file changes, asynchronous distributed execution of optimizing tasks,
+  transparent and imperceptible to users
+- Resource Isolation and Sharing — Allow resources to be isolated and shared at the table level, as well as setting resource quotas
+- Flexible and Scalable Deployment — Optimizers support various deployment methods and convenient scaling
+
+
+# Iceberg Format
+
+Apache Amoro supports all catalog types supported by Iceberg, including but not limited to: Hadoop, Hive, Glue, JDBC, Nessie, Snowflake, and so on.
```

Comment:
since this mentions **all**, does that also include the REST catalog?

Reply:
sure,rest catalog include it.